CHANGELOG.md

```diff
@@ -4,7 +4,11 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Unreleased
-- Ensure i18n resources are always initialized.
+
+
+### [1.0.3] - 2023-01-01
```

Comment:
I believe this should be left inside unreleased, it’s updated as part of the release process I think.

Right, @preetkaran20 ?

Reply:
Disregard.